### PR TITLE
FIX intent bug on actionState NONE

### PR DESCRIPTION
**Pull request by lawranceq42**
https://trello.com/c/myoWYcB1/172-bug-suzanne-intent-on-actionstate-none

### CODE REVIEW

File: webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts [hunk 1]
```diff
@@ -510,3 +510,7 @@ function onMyPlayerMoved(response: ResponseData, gameState:GameState, movedState
   }
 }
 
+function isReadyForTurn (response:ResponseData, gameState:GameState): boolean {
```
Comment (Frank1234) [inline webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts:513]: response is not needed, there is a session in gameState.

File: webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts [hunk 2]
```diff
@@ -510,3 +510,7 @@ function onMyPlayerMoved(response: ResponseData, gameState:GameState, movedState
...
+  return isMyTurn(gameState) && response.sessionState.actionState !== "NONE"
+}
+
```
Comment (Frank1234) [inline webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts:516]: please also fix it in initBadInputHandler

**Review by Frank1234 — CHANGES_REQUESTED** [review]
approved with one comment

File: webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts (hunk 2 above)
Comment (lawranceq42) [reply webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts:516]: done

File: webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts (hunk 1 above)
Comment (lawranceq42) [reply webhooks/ActionsOnGoogleFulfillment/functions/src/intentHandler.ts]: done

**Review by Frank1234 — APPROVED** [review]

**Pull request merged**